lib/symlink.js

```diff
@@ -31,7 +31,7 @@ function buildLinkPaths(libraries, symlinkPath) {
 function symLink(paths, callback) {
   try {
     paths.forEach(function(p) {
-      if (fs.lstatSync(p[0]).isDirectory()){
+      if (fs.lstatSync(p[0]).isDirectory() || fs.lstatSync(p[0]).isSymbolicLink()){
```

Comment:
`fs.lstatSync(p[0])` is probably not free (takes some meaningful time), so I recommend [memoizing](https://github.com/mapbox/cpp/blob/master/glossary.md#memoization) this call by calculating it once (`fs.lstatSync(p[0])`) and then re-using the value.

Reply:
I've updated `fs.lstatSync(p[0])` so it's only calculated once on this line, I'm not sure the best way to pass `fs.lstatSync(p[0])` into 'filterFunc' so it doesn't have to be recalculated again. `filter` is an options param for 'copySync` and it requires a function that only takes `src` and `dst` filepaths as an argument.

Reply:
Ah, nice spot that you could avoid another stat call if you could pass it into the `filter` function. Two ideas come to mind to do that:

1. Attach a property to the filterFunc dynamically (icky)
2. Bring the `filterFunc` into the scope of the `forEach` so that it could get access to the scope that has the `stat` you would create there. (a little less icky)
3. There might be a better way, I'm not thinking of that folks in the #javascript channel could offfer